Statistics.Test/StatsUnitTest.cs

```diff
@@ -9,20 +10,20 @@ public class StatsUnitTest
         [Fact]
         public void ReportsAverageMinMax()
         {
-            var statsComputer = new StatsComputer();
-            var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{1.5, 8.9, 3.2, 4.5});
+            //var statsComputer = new StatsComputer();
+            var computedStats = StatsComputer.CalculateStatistics(
+                new List<double>{1.5, 8.9, 3.2, 4.5});
             float epsilon = 0.001F;
-            Assert.True(Math.Abs(statsComputer.average - 4.525) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.max - 8.9) <= epsilon);
-            Assert.True(Math.Abs(statsComputer.min - 1.5) <= epsilon);
+            Assert.True(Math.Abs(computedStats.average - 4.525) <= epsilon);
+            Assert.True(Math.Abs(computedStats.max - 8.9) <= epsilon);
+            Assert.True(Math.Abs(computedStats.min - 1.5) <= epsilon);
         }
         [Fact]
         public void ReportsNaNForEmptyInput()
         {
-            var statsComputer = new StatsComputer();
-            var computedStats = statsComputer.CalculateStatistics(
-                new List<___>{});
+            //var statsComputer = new StatsComputer();
+            var computedStats = StatsComputer.CalculateStatistics(
+                new List<double>{});
```

Comment:
This test needs to assert that NaN is reported when the input is empty. Add such an assert.